cymetric/filters.py

```diff
@@ -1,6 +1,7 @@
 import warnings
 import pandas as pd
 import numpy as np
+from functools import reduce
```

Comment:
Why this new import?

Reply:
my linter (disabled now) wanted that....